_pages/ato/checklist.md

```diff
@@ -43,6 +40,8 @@ You are welcome to ask any questions as comments in the issue or #infrastructure
         1. [ ] Add the issue-free scan report to [the `ATOs` folder in Google Drive](https://drive.google.com/a/gsa.gov/folderview?id=0BynIxtx-CfkdckljM3BPSkdQT1U&usp=sharing)
 1. [ ] Update relevant documentation, primarily the README
 1. [ ] Add a [System Security Plan](https://pages.18f.gov/before-you-ship/ato/ssp/#template) to the repository
```

Comment:
My understanding was that the existing SSP process will be used until the Infrastructure Leads program was fully ramped up.